tests/unit/test_data_interface.py

```diff
@@ -267,3 +282,59 @@ def testSearchReferences(self):
                 for repoReference, reference in utils.zipLists(
                         repoReferences, references):
                     self._assertEqual(repoReference, reference)
+
+    def testSearchRnaQuantificationSets(self):
+        self._testSearchMethodInContainer(
+            'getRnaQuantificationSets',
+            self._client.search_rna_quantification_sets,
+            self._repo.getDatasets())
+
+    def testSearchRnaQuantifications(self):
+        # TODO Can't use because of weird search_rna_quantifications signature
```

Comment:
good catch - this is an artifact of an older version of the code where rnaQuantification was the top level